src/App.tsx

```diff
@@ -17,18 +72,32 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSelectByCompletedChange={(option) => setSelectByCompleted(option)}
```

Comment:
do you need to create wrappers here while passing the link to the function? can you just pass `setSelectByCompleted` function here?